src/inc/msquic.h

```diff
@@ -1605,6 +1605,35 @@ QUIC_STATUS
     _In_opt_ void* ClientSendContext
     );
 
+//
+// Connection Pool API
+//
+
+//
+// Creates a simple connection pool with NumberOfConnections connections
+// all with the same Context and Handler, and puts them in the
+// caller-supplied array.
+//
+// One of ServerName or ServerAddress *MUST* be supplied, and for the pool to
+// work correctly, the connections *MUST* be started with the same ServerName
+// and ServerPort as supplied in this call.
+//
+typedef
+_IRQL_requires_max_(PASSIVE_LEVEL)
+_Check_return_
+QUIC_STATUS
+(QUIC_API * QUIC_CONN_POOL_CREATE_FN)(
+    _In_ HQUIC Registration,
+    _In_ QUIC_CONNECTION_CALLBACK_HANDLER Handler,
+    _In_opt_ void* Context,
```

Comment:
They probably need to pass in an array of context pointers, a unique one for each.